docs/rrule.rst

```diff
@@ -5,6 +5,10 @@ rrule
 .. automodule:: dateutil.rrule
    :members:
    :undoc-members:
+   
+   .. autoclass:: dateutil.rrule.rrulebase
```

Comment:
This isn't actually a public function, so it shouldn't be showing up in the documentation.